elements/rh-button/src/rh-button.js

```diff
@@ -37,7 +37,10 @@ class RhButton extends RHElement {
 
   constructor() {
     super(RhButton.tag);
+    console.log("creating rh-button");
```

Comment:
@mwcz - Can you remove this console.log statement?

Reply:
Don't you mean *may* I remove this console.log statement?

Reply:
Yes...